MyFridge/MyFridge/Program.cs

```diff
@@ -0,0 +1,176 @@
+﻿namespace MyFridge
+{
+    internal class Program
+    {
+        private static void PrintByAvailabilityFrige(List<Refrigerator>refrigerators)
+        {
+            SortByAvailability(refrigerators);
+            foreach (var item in refrigerators)
+            {
+                Console.WriteLine(item.ToString());
+            }
+        }
+        private static void PrintByAvailabilityShelfe(Refrigerator refrigerator)
+        {
+            foreach (var item in refrigerator.SortByAvailability())
+            {
+                Console.WriteLine(item.ToString());
+            }
+        }
+        private static void PrintEExpiryDate(Refrigerator refrigerator)
+        {
+            foreach (var item in refrigerator.SortOfExpiryDate())
+            {
+                Console.WriteLine(item.ToString());
+            }
+        }
+        public static void WantToEat(Refrigerator refrigerator)
+        {
+            try
+            {
+            Console.WriteLine("what do you want to eat? enter: kind(Food Drink) & kosher(Meaty Dairy Fur )");
```

Comment:
readability: note the console is asking the user to enter Food or Drink but actually expected to int values as 0 or 1. Please phrase a more clear command.